unicode_gen.c

```diff
@@ -1239,6 +1277,11 @@ void build_conv_table(CCInfo *tab)
                     te->ext_len = 1;
                     te->ext_data[0] = te->data;
                     break;
+                case RUN_TYPE_L:
+                    te->type = RUN_TYPE_L_EXT;
+                    te->ext_len = 1;
+                    te->ext_data[0] = te->data;
+                    break;
```

Comment:
With some clever rearranging of the enum values, you can avoid code duplication by folding the three cases into one:
```c
case RUN_TYPE_L:
case RUN_TYPE_LF:
case RUN_TYPE_U:
    te->type += RUN_TYPE_L_EXT - RUN_TYPE_L;
    te->ext_len = 1;
    te->ext_data[0] = te->data;
    break;
```